### ISSUE TITLE
3.0.5 depends on vulnerable version of json5

### ISSUE THREAD

**Issue by louisch** [mathe42/vite-plugin-comlink#113]
Hi, I think this is possibly fixed on `main` already, but apparently version 3.0.5 of this plugin depends on json5 2.0.0 - 2.2.1, which is a "high severity vulnerability"? (`npm audit fix --force` causes this plugin to downgrade to 3.0.2, which is not ideal)

```
json5  2.0.0 - 2.2.1
Severity: high
Prototype Pollution in JSON5 via Parse Method - https://github.com/advisories/GHSA-9c47-m6qq-7p4h
fix available via `npm audit fix --force`
Will install vite-plugin-comlink@3.0.2, which is a breaking change
node_modules/vite-plugin-comlink/node_modules/json5
  vite-plugin-comlink  >=3.0.3
  Depends on vulnerable versions of json5
  node_modules/vite-plugin-comlink
```

**Comment by Tsury** [comment]
Seems like it was fixed on `main` about 9 months ago - I hope this repo is not abandoned, it's pretty useful.

**Comment by prazii** [comment]
@mathe42, any updates on this? Thanks!

**Comment by mathe42** [comment]
fixed with 4.0
